pkg/api/admin/openshiftcluster.go

```diff
@@ -179,23 +179,87 @@ const (
 	VMSizeStandardD2sV3 VMSize = "Standard_D2s_v3"
 
 	VMSizeStandardD4asV4  VMSize = "Standard_D4as_v4"
+	VMSizeStandardD4asV5  VMSize = "Standard_D4as_v5"
 	VMSizeStandardD8asV4  VMSize = "Standard_D8as_v4"
+	VMSizeStandardD8asV5  VMSize = "Standard_D8as_v5"
 	VMSizeStandardD16asV4 VMSize = "Standard_D16as_v4"
+	VMSizeStandardD16asV5 VMSize = "Standard_D16as_v5"
 	VMSizeStandardD32asV4 VMSize = "Standard_D32as_v4"
+	VMSizeStandardD32asV5 VMSize = "Standard_D32as_v5"
 
 	VMSizeStandardD4sV3  VMSize = "Standard_D4s_v3"
+	VMSizeStandardD4sV4  VMSize = "Standard_D4s_v4"
+	VMSizeStandardD4sV5  VMSize = "Standard_D4s_v5"
 	VMSizeStandardD8sV3  VMSize = "Standard_D8s_v3"
+	VMSizeStandardD8sV4  VMSize = "Standard_D8s_v4"
+	VMSizeStandardD8sV5  VMSize = "Standard_D8s_v5"
 	VMSizeStandardD16sV3 VMSize = "Standard_D16s_v3"
+	VMSizeStandardD16sV4 VMSize = "Standard_D16s_v4"
+	VMSizeStandardD16sV5 VMSize = "Standard_D16s_v5"
 	VMSizeStandardD32sV3 VMSize = "Standard_D32s_v3"
+	VMSizeStandardD32sV4 VMSize = "Standard_D32s_v4"
+	VMSizeStandardD32sV5 VMSize = "Standard_D32s_v5"
+	VMSizeStandardD64sV5 VMSize = "Standard_D64s_v5"
+	VMSizeStandardD96sV5 VMSize = "Standard_D96s_v5"
+
+	VMSizeStandardE8V4  VMSize = "Standard_E8_v4"
```

Comment:
The VM SKUs from here to line 219 do not support Premium disk types [[ref](https://learn.microsoft.com/en-us/azure/virtual-machines/vm-naming-conventions#naming-convention-explanation)]. Have we verified that these create with standard disks correctly? The [function I added](https://github.com/Azure/ARO-RP/blob/902ae3f3b79ab39dfc60b96c31346c6c8bddd892/pkg/util/computeskus/computeskus.go#L87-L93) *should* be using the capabilities reported by the SKU itself but it would be nice to get confirmation.

Reply:
@dem4gus I need to test a few more VM families still (including this one) - I'll update the PR when I have confirmation that things are working well.

Reply:
@dem4gus as of now, instances that don't support PremiumIO are not installing correctly if selected for master due to this code: https://github.com/Azure/ARO-RP/blob/master/pkg/installer/deployresources_resources.go#L123-L125 && https://github.com/Azure/ARO-RP/blob/master/pkg/installer/deployresources_resources.go#L194-L196 - so even though we were setting this correctly in install config, the master and bootstrap machines were still hardcoded to use Premium.

OCP doesn't technically support non-Premium instance types anyway, so I have removed them in this PR.